HOWTO.md

```diff
@@ -118,11 +118,9 @@ wait for it to complete downloading the blockchain.
 
 Write this in `bitcoin.conf`:
 
-    rpcuser=<rpc-username>
```

Comment:
This is no longer necessary if both programs run on the same server. If not, the new rpcauth option should be used:
https://github.com/bitcoin/bitcoin/blob/master/contrib/debian/examples/bitcoin.conf

Rpcuser/rpcpassword is deprecated and will be removed in a future release.